src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -81,18 +94,33 @@ export default function NotificationsPane({
           ) : (
             notifications.map(notification => {
               const notificationType = notification?.message_type;
+              const mergedIndividualGuid =
+                notification?.message_values?.target_individual_guid;
+
               const currentNotificationSchema = get(
                 notificationSchema,
                 notificationType,
               );
               const showNotificationDialog =
                 currentNotificationSchema?.showNotificationDialog &&
                 refreshNotifications !== undefined;
-              const senderName = get(
-                notification,
-                'sender_name',
-                'Unnamed User',
+              const {
+                senderName,
+                user1Name,
+                user2Name,
+                yourIndividualName,
+                theirIndividualName,
+                formattedDeadline,
+              } = getNotificationProps(notification);
+              const deriveButtonPath = get(
+                currentNotificationSchema,
+                'deriveButtonPath',
               );
+              let customButtonPath;
```

Comment:
why is this `let`? seems like you could just do 
```
const customButtonPath = deriveButtonPath && deriveButtonPath(mergedIndividualGuid)
```

Reply:
Done. Wom wom. Thanks.